src/components/Auth/Login.js

```diff
@@ -1,19 +1,177 @@
+  border-bottom: 1px solid #eee;
+  width: 343px;
+`;
+
+const LoginOptionButton = styled.button`
+  min-width: 167px;
+  max-width: 176px;
+  padding: 12px 40.5px;
+  background: none;
+  font-size: 16px;
+  font-weight: ${({ isClicked }) => (isClicked ? 'bold' : 'normal')};
+  text-align: center;
+`;
+
+const LoginOptionMenuBar = styled.div`
+  display: block;
+  position: absolute;
+  width: ${({ location }) => (location === 1 ? 167 : 176)}px;
+  height: 1px;
+  background-color: #222;
+  transition: transform 0.2s ease;
+  transform: translateX(${({ location }) => (location === 1 ? 0 : 167)}px);
+`;
+
+const LoginForm = styled.form`
+  display: ${({ display }) => (!display ? 'none' : 'block')};
+  position: relative;
+  width: 100%;
+`;
+
+const StyledInput = styled(S.StyledInput)`
+  & + & {
+    margin-top: 0;
+  }
+`;
+
+const StyledOptionLink = styled(S.StyledLink)`
+  padding: 0 15px;
+  height: 12px;
+  :nth-child(n) {
+    border-right: 1px solid #999;
+  }
+  :last-child {
+    border: none;
+  }
+`;
+
+const StyledInputContainer = styled.div`
+  margin-bottom: 16px;
+  position: relative;
+  :nth-child(2) {
+    margin-bottom: 8px;
+  }
+`;
+
+const PwdSee = styled.span`
+  display: block;
+  right: 16px;
+  top: 14px;
+  position: absolute;
+  cursor: pointer;
+`;
+const EyeImg = styled.img`
+  width: 24px;
+  height: 24px;
+`;
+
+const SNSLoginOptionSection = styled.section`
+  display: ${({ display }) => (display ? 'none' : 'flex')};
+  width: 100%;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+`;
+
+const GoogleLoginButton = styled.button`
+  width: 343px;
+  height: 44px;
+  margin-bottom: 16px;
+  padding: 0 10px;
+  border: solid 0.5px #c4c4c4;
+  background-color: #fff;
+
+  font-size: 16px;
+  font-weight: normal;
+  text-align: center;
+  color: #000;
+
+  background-image: url('/asset/google-logo.svg');
+  background-position-y: center;
+  background-position-x: 12px;
+  background-repeat: no-repeat;
+
+  :after {
+    content: '구글 로그인';
+  }
+`;
+const NaverLoginButton = styled.button`
+  width: 343px;
+  height: 44px;
+  margin-bottom: 16px;
+  padding: 0 0 4px;
+  background-color: #03c75a;
+
+  font-size: 16px;
+  font-weight: normal;
+  text-align: center;
+  color: #fff;
+
+  background-image: url('/asset/naver-logo.svg');
+  background-position-y: center;
+  background-position-x: 12px;
+  background-repeat: no-repeat;
+  :after {
+    content: '네이버 로그인';
+  }
+`;
+const KakaoLoginButton = styled.button`
+  width: 343px;
+  height: 44px;
+  padding: 0 10px;
+  background-color: #fee500;
+
+  font-size: 16px;
+  font-weight: normal;
+  text-align: center;
+  color: #000;
+
+  background-image: url('/asset/kakao-logo.svg');
+  background-position-y: center;
+  background-position-x: 12px;
+  background-repeat: no-repeat;
+
+  :after {
+    content: '카카오 로그인';
+  }
+`;
+
+/**
+ * TODO:
+ * - [] 비회원으로 이용하기 클릭시, keyword리스트 페이지로 이동
+ * - [] 구글 로그인
+ * - [] 네이버 로그인
+ * - [] 카카오 로그인
+ * @returns

```

Comment:
오홍... 주석 좋네요 
다만 `@returns` 는 반환값 기입 하지 않으실거라면 지워주셔도 좋을것 같습니다~

```suggestion
/**
 * TODO:
 * - [] 비회원으로 이용하기 클릭시, keyword리스트 페이지로 이동
 * - [] 구글 로그인
 * - [] 네이버 로그인
 * - [] 카카오 로그인
```